CHANGELOG.md

```diff
@@ -553,6 +553,7 @@
 - Budgets do not collect IATI fields or currency as they are set by default
 - Budgets tables do not show IATI fields and only show the financial year
 - Budgets funding type must be the same as the parent activity
+- Make recieving organisation optional
```

Comment:
If you're amending / fixing conflicts, would you mind fixing the typo here too? "recieving" --> "receiving")

Reply:
Done!